LaborasLangCompiler/Codegen/MetadataHelpers.cs

```diff
@@ -150,6 +162,11 @@ public static bool IsAssignableTo(this TypeReference right, TypeReference left)
                 return true;
             }
 
+            if(right.IsNullType() && !left.IsValueType)
+            {
+                return true;
```

Comment:
this should be:

if (left.IsNullType())
    return false;

if (right.IsNullType())
    return !left.IsValueType;